brave/brave-core
# [Brave News]: Don't apply the old background color update for new styles

Maybe resolves https://github.com/brave/brave-browser/issues/26366

Note: I can't reproduce the issue in my dev build (but I do see it in Beta & Nightly), so this fix is purely speculative. I've tried both release & official builds so I'm not sure why.

My theory as to why this is happening is that we were painting the background, and then repainting it as transparent, which was messing up the anti aliasing of the text. Not painting the original background color should fix it.

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Enable the brave://flags/#brave-news-subscribe-button flag
2. Navigate to https://theatlantic.com/
3. When the Brave News Subscribe button appears in the toolbar, the text should be legible for `Follow`, `Unfollow` and `Manage Feeds`
